content/xr_design_and_accessibility.mdx

```diff
@@ -0,0 +1,53 @@
+
+Under certain conditions, accessibility can be the difference between a user having an experience and a user being excluded from one. Sometimes these disabilities can be permanent, but they can also be situational or temporary. For example, a person without an arm (a permanent disability), a person with a cast (a temporary disability) and a person holding a child (a situational disability) might benefit from the opportunity to choose their preference of using one VR controller rather than two without feeling limited by the experience. 
+
+Although we often consider the ideal user, most people do not strictly fall into these categories. Allowing people to choose how they experience their experiences does not only benefit people with disabilities, it benefits everyone. While there may be the inclination to make your experience a literal extended version of reality, we also have the opportunity to create an extended reality that adapts to the needs of the people living in it. People will need different options based on their needs. The ability to adjust preset options allows the user to experience your application in a manner that is comfortable to them. 
+
+This is not a clear cut guide of the do’s and don’ts when designing user experiences in VR, but  a compilation of our research and resources on the subject. With everything, test and see if it works for you and your needs.  
+
+# Text
+|| <div style="width:250px">Options</div> | Why |
+|:-----|:-----:|-----:|
+| Color | Light-colored font with an outline | Backgrounds are more likely to be dark than light, so light colors contrast best against dark backgrounds. Test different colored fonts according to your needs. |
+| Location | <li>At a fixed distance in front of the user</li> <li>On a fixed location</li> <li>Above a character’s head</li> | What does the text convey and what location will best affect the user’s experience? |
+| Font size | <li>Small</li><li>Medium</li><li>Large</li> | Set the default size to medium and allow the user to adjust font size (either by creating preset options or scale) to their comfort level.  This should start at 10% of the screen’s height and adjust as needed. |
+| Friendly fonts | <li>Arial</li> <li>OpenDyslexic</li> | Sans serif and monospaced fonts are often easier to read. Allowing a user to choose a preferred font based on their needs means that their experience is less likely to be diminished by something that we (the devs) can easily adjust. |
+| Visual indicators | <li>Arrows</li> <li>Animations</li> | Due to the nature of the 360 degree experience, we often lack control over what the user is seeing. If there is text that needs to be read, cue the user with visual indicators if they’re looking in the wrong direction. |
+| Alignment | Left-aligned and no justification | Left-aligned text with no justification is easier to read for those with dyslexia. |
+
+# Subtitles
```

Comment:
Would it be more accurate to title this section "Closed Captioning"?

Reply:
Totally. Will update!